Week-3/Responsiveness/Bootstrap for a vacation website/script.js

```diff
@@ -0,0 +1,12 @@
+function showAlert(alertId) {
+   const bootstrapAlert = document.querySelector("#" + alertId);
```

Comment:
this looks like ID, you can use document.getElementById

Reply:
This was the default file in script.js that was already provided to us.